Planner/app/src/main/java/rdatu/android/cyscorpions/com/projectplanner/view/ListPlannerActivity.java

```diff
@@ -0,0 +1,171 @@
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import rdatu.android.cyscorpions.com.projectplanner.R;
+import rdatu.android.cyscorpions.com.projectplanner.controller.PlannerPagerAdapter;
+
+/**
+ * Created by rayeldatu on 7/27/15.
+ */
+public class ListPlannerActivity extends FragmentActivity implements ListPlannerFragment.Callbacks, DatePickerDialog.Callbacks {
+
+    public static final String FUNCTION_FORCHANGE = "jumpto";
+    public static final String EXTRA_TIME_SELECTED = "timeselected";
+    public static final String EXTRA_DATE_SELECTED = "dateselected";
+    public static final String EXTRA_NEWDATE = "newdate";
+    public static final String EXTRA_TASKNAME = "name";
+    public static final String EXTRA_DESC = "description";
+    public static final String EXTRA_PLACE = "place";
+    public static final String EXTRA_PRIORITY = "priority";
+    private static final int PAGE_MIDDLE = 1;
+    final ListPlannerFragment[] LIST_PLANNER = new ListPlannerFragment[3];
+    private ViewPager mViewPager;
+    private Calendar mCurrentCalendar;
+    private int mSelectedPageIndex;
+    private PlannerPagerAdapter mAdapter;
+
+
+    @Override
+    public void onStart() {
+        super.onStart();
+
+    }
+
+    @Override
+    @SuppressWarnings("deprecation")
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Log.d("Planner", "OnCreate");
+
+        Calendar prevDay, nextDay;
+
+        mViewPager = new ViewPager(this);
+        mViewPager.setId(R.id.ViewPager);
+
+        if (getIntent().hasExtra(EXTRA_NEWDATE)) {
+            SimpleDateFormat df = new SimpleDateFormat("MMM-dd-yyyy");
+            try {
+                mCurrentCalendar = Calendar.getInstance();
+                mCurrentCalendar.setTime(df.parse(getIntent().getStringExtra(EXTRA_NEWDATE)));
+                mCurrentCalendar.add(Calendar.DAY_OF_MONTH, -1);
+            } catch (Exception e) {
+
+            }
+        } else {
+            mCurrentCalendar = Calendar.getInstance();
+            mCurrentCalendar.add(Calendar.DAY_OF_MONTH, -1);
+        }
+
+        setContentView(mViewPager);
+
+        prevDay = (Calendar) mCurrentCalendar.clone();
+        nextDay = (Calendar) mCurrentCalendar.clone();
+
+        prevDay.add(Calendar.DAY_OF_MONTH, -1);
+        nextDay.add(Calendar.DAY_OF_MONTH, 1);
+
+        LIST_PLANNER[0] = ListPlannerFragment.newInstance(prevDay, getApplicationContext());
+        LIST_PLANNER[1] = ListPlannerFragment.newInstance(mCurrentCalendar, getApplicationContext());
+        LIST_PLANNER[2] = ListPlannerFragment.newInstance(nextDay, getApplicationContext());
+
+        mAdapter = new PlannerPagerAdapter(getSupportFragmentManager(), LIST_PLANNER);
+
+        mViewPager.setOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+            @Override
+            public void onPageScrolled(int i, float v, int i1) {
+                //Intentionally left Blank!
```

Comment:
Will result to an error po ma'am these methods needs to be overridden po. eventhough walang implementation i believe the method is abstract po.